Makefile

```
@@ -1,6 +1,6 @@
 #Source: https://github.com/evanugarte/mongocxx-tutorial/blob/09dc4bf76d57fe40cf7154a8eb9e7530d49ab536/Makefile
 CXX		  := g++
-CXX_FLAGS := -Wall -Wextra -std=c++17 -ggdb
+CXX_FLAGS := -Wall -Wextra -std=c++17 -ggdb -Wno-unused-parameter -Wno-deprecated-declarations
```

Suppressing errors because our translator REST API requires an MD5 hash, and the example provided was deprecated. It's functional, but it's kinda yucky -- not sure how to work around this without changing the REST endpoint.